packages/protocol-kit/src/types/safeProvider.ts

```diff
@@ -11,12 +14,12 @@ export type HexAddress = string
 export type PrivateKey = string
 export type HttpTransport = string
 export type SocketTransport = string
-export type SafeSigner = HexAddress | PrivateKey
+export type SafeSigner = HexAddress | PrivateKey | PasskeySigner
 
 export type SafeProviderConfig = {
   /** signerOrProvider - Ethers signer or provider */
   provider: Eip1193Provider | HttpTransport | SocketTransport
-  signer?: HexAddress | PrivateKey
+  signer?: HexAddress | PrivateKey | PasskeySigner | PasskeyArgType
```

Comment:
Confused here with the added `PasskeySigner`. This is to instantiate the provider with something from outside as an address, a privateKey or a passkey args object. Can this be instantiate with the PasskeySigner class?

In any case this should be added as weel to the SafeSigner type above that now is being used in multiple places. I would prefer to continue using SafeProviderConfig['signer'] as well